instance-listener/main.go

```diff
@@ -90,7 +90,7 @@ func main() {
 				log.Printf("ERROR: broadcast-ip: %v; failed writing to broadcast udp socket: "+err.Error(), BROADCAST_IPv4)
 				return
 			}
-			time.Sleep(1000 * time.Millisecond)
+			time.Sleep(5000 * time.Millisecond)
```

Comment:
what's the purpose for this change?

Reply:
The listener was using 100% of CPU and stop functioning properly after a while (something like a day). I resolved this by extending the sleep time, which also mean that it send less broadcast on the network. The server I was testing on was a bit old and I should test again on a recent server with a better CPU.

Reply:
I did the test again on a Xeon E5 1650 v2 3.50 GHz. With the sleep set to 1 second, the listener is using 3.25 GHz. If the sleep is set to 5 seconds the CPU usage drop to 770 MHz.

Reply:
I've noticed listener spiking CPU on `virtualbox` too. Something seems a bit fishy 🤔

Reply:
probably a rumprun issue...